packages/affiliates/gas-rebate/VoterGasRebateV2.ts

```diff
@@ -43,7 +68,7 @@ export async function run(): Promise<void> {
   console.log("Previous Month End:", moment(prevMonthEnd).format(), "& block", toBlock);
 
   // Fetch all commit and reveal events.
-  const voting = await ethers.getContractAt("VotingV2", await getAddress("VotingV2", 1));
+  const voting = await (hre as any).ethers.getContractAt("VotingV2", await getAddress("VotingV2", 1));
```

Comment:
do we need any casting even with `"@nomiclabs/hardhat-ethers"` added?

Reply:
It can be removed, thanks!